commands/define.js

```diff
@@ -0,0 +1,57 @@
+const { errorColor, successColor } = require("../config.json")
+const Discord = require("discord.js")
+
+module.exports = {
+    name: "define",
+    description: "Gives you the definition of a word in English along with some details.",
+    usage: "+define <query>",
+    cooldown: 10,
+    allowDM: true,
+    channelWhitelist: ["549894938712866816", "624881429834366986", "730042612647723058", "749391414600925335", "551693960913879071"], // bots staff-bots bot-development bot-translators admin-bots
+    execute(message, strings, args) {
+        message.channel.startTyping()
```

Comment:
This should be on the function since that may not run in case there is an error, in which case the bot will be stuck typing

Reply:
This should also do the trick.